packages/server/src/config.ts

```diff
@@ -55,6 +55,7 @@ export interface MedplumServerConfig {
   heartbeatEnabled?: boolean;
   accurateCountThreshold: number;
   slowQueryThresholdMilliseconds?: number;
+  slowQuerySampleRate?: number;
```

Comment:
Need to add numeric param handling in `packages/server/src/cloud/aws/config.ts`

Reply:
Apologies in advance, because that is some unfortuante tech debt.  Someday we should figure out a proper solution for that.